pyscriptjs/src/plugin.ts

```diff
@@ -2,8 +2,10 @@ import type { AppConfig } from './pyconfig';
 import type { Interpreter } from './interpreter';
 import type { UserError } from './exceptions';
 import { getLogger } from './logger';
+import { make_PyScript } from './components/pyscript';
 
 const logger = getLogger('plugin');
+type PyScriptTag =  InstanceType<ReturnType<typeof make_PyScript>>;
```

Comment:
Nice! I've been trying to figure out how to get the typing for this to work haha

Reply:
Nifty right?? I've been working my way through [the TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/intro.html) and this jumped out as useful.